prototype_source/lite_interpreter.rst

```diff
@@ -0,0 +1,229 @@
+(Prototype) Introduce lite interpreter workflow in Android and iOS
+==================================================================
+
+**Author**: `Chen Lai <https://github.com/cccclai>`_, `Martin Yuan <https://github.com/iseeyuan>`_
+
+Introduction
+------------
+
+This tutorial introduces the steps to use lite interpreter on iOS and Android. We'll be using the ImageSegmentation demo app as an example. Since lite interpreter is currently in the prototype stage, a custom pytorch binary from source is required.
+
+
+To get ImageSegmenation demo app,
+
+**iOS**: https://github.com/pytorch/ios-demo-app/tree/master/ImageSegmentation
+
```

Comment:
Missing the Android link?

Reply:
Oh good catch. I should remove this line. I move the demo link to the corresponding Android/iOS session.